packages/circuits/tests/common.ts

```diff
@@ -10,3 +13,55 @@ export const circomkit = new Circomkit({
     ...config,
     verbose: false
 })
+
+/**
+ * Returns a BabyJub-compatible random value. We create it by first generating
+ * a random value (initially 256 bits large) modulo the snark field size as
+ * described in EIP197. This results in a key size of roughly 253 bits and no
+ * more than 254 bits. To prevent modulo bias, we then use this efficient
+ * algorithm:
+ * http://cvsweb.openbsd.org/cgi-bin/cvsweb/~checkout~/src/lib/libc/crypt/arc4random_uniform.c
+ * @returns A BabyJub-compatible random value.
+ */
+export const genRandomBabyJubValue = (): bigint => {
```

Comment:
Another note: these methods are duplicated from @zk-kit/poseidon-cipher test utils. We should move things in a common space in the future.

Reply:
`@zk-kit/utils` sounds like a good place to move them IMO.

We can add a task for this.